management-center/start.sh

```diff
@@ -12,12 +12,10 @@ if [ "x$MAX_HEAP_SIZE" != "x" ]; then
 	JAVA_OPTS="$JAVA_OPTS -Xmx${MAX_HEAP_SIZE}"
 fi
 
-export CLASSPATH=$HAZELCAST_HOME/*:$CLASSPATH/*
-
 echo "########################################"
 echo "# RUN_JAVA=$RUN_JAVA"
 echo "# JAVA_OPTS=$JAVA_OPTS"
 echo "# starting now...."
 echo "########################################"
 
-java -server $JAVA_OPTS -Dhazelcast.mancenter.home=$MANCENTER_DATA -jar mancenter-$HZ_VERSION.war
+java -server $JAVA_OPTS -Dhazelcast.mancenter.home=$MANCENTER_DATA -cp mancenter-$HZ_VERSION.war:$HAZELCAST_HOME/*:$CLASSPATH_ARG/* Launcher 8080 mancenter
```

Comment:
Why do we need `$HAZELCAST_HOME/*` here? `mancenter.war` file and the extra classpath directory you've added (`/opt/lib`) should be enough for anyone who requires an extra JAR on the classpath.